### PR TITLE
Stabalized Tree with Springs and Added Wind

### PR DESCRIPTION
**Pull request by Okaghana**
Added some Strings between the Joints of the Tree and the Position where they originally were created. This pulls the Joints always back to the original position and so stabilize the Structure of the tree. The Strength of the Springs decreases further out.

Also add randomly occurring wind that pushes the tree from left to right

Unfortunately the outermost branches are quite bouncy when the wind begins, which can probably be resolved by not just starting the wind abruptly, but instead letting the wind slowly build up

If one wants the tree bouncier just decrease the strength of the Springs I added

~Okaghana

### CODE REVIEW

File: sketch.js [hunk 1]
```diff
@@ -72,6 +75,19 @@ function draw() {
...
+  
+  // Randomly Create wind
+  if (random() < 0.01){					// optionally add "&& wind == 0" to only create wind when the previous wind disappeared
+	  wind = random(-1, 1) * 35
+  }
+  
+  // Decrease Wind
+  if (wind != 0){
+	wind *= 0.98
+	if ((wind > 0 && wind < 0.5) || (wind < 0 && wind > -9.5)){wind = 0}
```
Comment (Okaghana) [inline sketch.js:87]: Typo Here. Should of course say "wind > -0.5" and not "wind > -9.5"

**Comment by shiffman** [comment]
Thank you for this! I haven't decided what to do exactly with this repo. Thinking out loud, I think it would be nice to accept people's pull requests, improve this project, and feature it on a future livestream. If / when the live stream is edited into a coding challenge video, the "original" as-is code can live on the website.

A good start would be to outline some information about the project, what kinds of contributions we're looking for, and information about #TeamTrees in the README file? 

This fits into the "make the tree more realistic / stable". I think maybe a goal that makes sense is to make a "beautiful" algorithmic tree swaying in the wind in a nature scene?

I'm open to inputs and suggestions!

**Comment by duskvirkus** [comment]
I think a good approach could be to make an home page that sits at the top level and provides navigation and then we could have sub directories with the original sketch as well as contributions and remixes. Then we could preserve the original sketch and also have additional sketches and we could just have it hosted through github pages. I'd be happy to work on that as well as a readme about how to contribute if it seems like a good solution.

**Comment by shiffman** [comment]
This sounds like a good plan to me @violetcraze, thank you! 🎄🌳🌴

**Comment by Okaghana** [comment]
@violetcraze @shiffman I'm currently further improving the Code to make the wind more realistic and will probably add that here when I'm satisfied with it. I'll maybe even implement a "Wind-Indicator" (like these wavy lines moving across the screen or some clouds). If you have other ideas: I'm open for suggestions 🌲🌳

**Comment by shiffman** [comment]
ok, please submit it here @Okaghana I'm excited to see it!

**Pull request closed without merging**